src/lib/yang/state.lua

```diff
@@ -23,14 +23,20 @@ local function flatten(val)
     return rtn
 end
 
-local function find_counters(pid)
+function find_counters(pid)
     local path = shm.root.."/"..pid..counter_directory
-    local counters = {}
+    local apps = {}
     for _, c in pairs(lib.files_in_directory(path)) do
+        local counters = {}
         local counterdir = "/"..pid..counter_directory.."/"..c
-        counters[c] = shm.open_frame(counterdir)
+        for k,v in pairs(shm.open_frame(counterdir)) do
```

Comment:
Wow, so these counters had some magical thing attached to them that made them print with commas???  Bizarre.